Tita-Clone/Tita-Clone/Controllers/LoginViewController.swift

```diff
@@ -9,28 +9,137 @@ import UIKit
 import SnapKit
 import Then
 
-class LoginViewController: UIViewController {
+class LoginViewController: UIViewController, UITextFieldDelegate {
     //MARK: - Properties
+    private let backImg = UIImageView().then {
+        $0.image = UIImage(named: "Tita-LoginVCBack")
+    }
+    
+    private let logoImg = UIImageView().then {
+        $0.image = UIImage(named: "Tita-Logo")
+    }
+    
+    private let welcomeLabel = UILabel().then {
+        $0.numberOfLines = 2
+        $0.textAlignment = .left
+        $0.text = "어서오세요,\nTimeTable입니다!"
+        $0.dynamicFont(fontSize: 27, currentFontName: "NotoSans-Bold")
+    }
+    
+    private let idContainer = TextFieldView().then {
+        $0.button.setTitle("아이디를 잊으셨나요?", for: .normal)
+        $0.button.setUnderline()
+        $0.button.addTarget(self, action: #selector(tapForgetId), for: .touchUpInside)
+    }
+    
+    private let pwContainer = TextFieldView().then {
+        $0.textField.placeholder = "비밀번호"
+        $0.textField.isSecureTextEntry = true
+        $0.button.setTitle("비밀번호를 잊으셨나요?", for: .normal)
+        $0.button.setUnderline()
+        $0.button.addTarget(self, action: #selector(tapForgetPw), for: .touchUpInside)
+    }
     
+    private let enterButton = UIButton().then {
+        $0.setTitle("입장하기", for: .normal)
+        $0.setTitleColor(.black, for: .normal)
+        $0.dynamicFont(fontSize: 18, currentFontName: "NotoSans-Bold")
+        $0.backgroundColor = .rgb(red: 255, green: 251, blue: 235)
+        $0.layer.cornerRadius = 10
+        $0.layer.applySketchShadow(color: .black, alpha: 0.25, x: 2, y: 2, blur: 10, spread: 0)
+        $0.clipsToBounds = false
+        $0.addTarget(self, action: #selector(tapEnter), for: .touchUpInside)
+    }
+    
+    private let signUpButton = UIButton().then {
+        $0.setTitle("혹시 아직 회원이 아니신가요?", for: .normal)
+        $0.setTitleColor(.black, for: .normal)
+        $0.dynamicFont(fontSize: 12, currentFontName: "NotoSans-Regular")
```

Comment:
```suggestion
        $0.dynamicFont(fontSize: 12, currentFontName: "NotoSans")
```
위와 같이 선언해야 font 적용이 됐던 것 같은데 확인이 필요할 것 같네요! 확인 후 적용하세요.

Reply:
확인했습니다!